getting-started/python/Client and Server Structure/README.md

```diff
@@ -0,0 +1,42 @@
+
+- Such role reversal is common in many systems, so, frequently, client-server systems could be more accurately described as peer-to-peer systems.
+
+## Objects & Proxies
+
+- An Ice object is an entity in the local or a remote address space that can respond to client requests. The objects are created by Server
+- Each Ice object has a unique object identity. An object's identity is an identifying value that distinguishes the object from all other objects. The Ice object model assumes that object identities are globally unique, that is, no two objects within an Ice communication domain can have the same object identity.
+
+- For a client to be able to contact an Ice object, the client must hold a proxy for the Ice object. A proxy is an artifact that is local to the client's address space; it represents the (possibly remote) Ice object for the client. A proxy acts as the local ambassador for an Ice object: when the client invokes an operation on the proxy, the Ice run time:
+    1. Locates the Ice object
+    2. Activates the Ice object's server if it is not running
+    3. Activates the Ice object within the server
+    4. Transmits any in-parameters to the Ice object
+    5. Waits for the operation to complete
+    6. Returns any out-parameters and the return value to the client (or throws an exception in case of an error)
+- A proxy encapsulates all the necessary information for this sequence of steps to take place.
+- You can find the object address in the `implements/etc/config`.
+- You can find the proxy address in the `requires/etc/config`.
+
+## Example of how the robocomp components uses ICE to share data
+
+- In this tutorial, We are going to learn how the different components in robocomp, share data using the Ice's peer-to-peer systems. 
+- For the data to be shared between components, we need to have at least 2 components i.e. One component will act as a server and the other will be the client.
+
+- We can use `implement` or `publish` keyword in the `Component Specific Definition Language(.cdsl)` to specify that the component will be implemented and generally this component will generate some kind of data that might be required by another component of robocomp.
+
+- We can use `require` or `subscribe` keywords in `Component Specific Definition Language(.cdsl)` to specify that the current component needs some information from the required component of the robocomp.
+
+- We will create a server and a client for CameraSimple component that will show you how you can share an Image data(Information) from client to server.
+
+- The implements folder contains implementation of the CameraSimple(will act as a Server). And the requires folder contains a component that requires the information send by CameraSimple component.
```

Comment:
And maybe a pair of links to the Readmes of the implements and requires folders to be able to follow the tutorial?